packages/ibm-products/src/components/ConditionBuilder/ConditionBlock/ConditionBlock.js

```diff
@@ -0,0 +1,247 @@
+import React, { useContext, useState } from 'react';
+import { Close } from '@carbon/react/icons';
+import { ConditionBuilderItem } from '../ConditionBuilderItem/ConditionBuilderItem';
+import { pkg } from '../../../settings';
+import PropTypes from 'prop-types';
+
+import cx from 'classnames';
+import ConditionConnector from '../ConditionConnector/ConditionConnector';
+
+import { ConditionBuilderContext } from '../ConditionBuilderContext/DataTreeContext';
+import { ConditionBuilderButton } from '../ConditionBuilderButton/ConditionBuilderButton';
+
+/**
+ * This component build each block of condition consisting of property, operator value and close button.
+ */
+const blockClass = `${pkg.prefix}--condition-builder`;
+
+function ConditionBlock(props) {
```

Comment:
as a convention we should continue to maintain the declaration like so

```
const ConditionBlock = ({ onRemove, state, ... ) => {})
```

Reply:
modified as advised